content/tracing/setup/cpp.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Tracing C++ Applications
+kind: Documentation
+aliases:
+- /tracing/cpp/
+- /tracing/languages/cpp
+further_reading:
+- link: "https://github.com/DataDog/dd-opentracing-cpp"
+  tag: "Github"
+  text: Source code
+- link: "tracing/visualization/"
+  tag: "Documentation"
+  text: "Explore your services, resources and traces"
+- link: "tracing/advanced_usage/?tab=c"
+  tag: "Advanced Usage"
+  text: "Advanced Usage"
+---
+
+<div class="alert alert-warning">
+The APM tracer for C++ applications is in Open Public Beta.
+</div>
+
+## Getting Started
+
+To begin tracing applications written in any language, first [install and configure the Datadog Agent](https://docs.datadoghq.com/tracing/setup).
+
+You will need to compile against [OpenTracing-cpp](https://github.com/opentracing/opentracing-cpp).
+
+## Compatibility
+
+dd-opentracing-cpp needs C++14 to build, but if you use [dynamic loading](#dynamic-loading) then you are instead only limited by OpenTracing's requirement for [C++11 or later](https://github.com/opentracing/opentracing-cpp/#cc98).
+
+Supported platforms are: Linux & Mac. If you need Windows support, please let us know.
+
+## Installation
+
+Datadog tracing can be enabled in one of two ways:
+
+* Compile against dd-opentracing-cpp, where the Datadog lib is compiled in and configured in code
```

Comment:
```suggestion
* Compile against `dd-opentracing-cpp`, where the Datadog lib is compiled and configured in code
```

Reply:
Maybe "compiled-in, and configured in code"? What I mean to say is that the library is included in compilation. Similar to saying that something is "baked in".

Entirely different phrasing if we want to work around it entirely:

 * Compile against `dd-opentracing-cpp`; including Datadog headers and linking against the library, and configuring the tracer in code